### PR TITLE
feature: added avatar

### PR DESCRIPTION
**Pull request by omgaikwad**
Screenshot (22)

Feedback for improvements is appreciated.

### CODE REVIEW

**Comment by t007rushi** [comment]
Naming Convention is good & comments readable. You can add border-color for variations one more It would be great if you have text-avatars just suggestions.

**Comment by faisal-kursheedali** [comment]
Your avatar.css file is looking good, But it is good to use the rem than px.

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/* Avatar */
+.avatar {
+  height: 50px;
```
Comment (shivampandey0) [inline components/avatar/avatar.css]: Try to use rem units

File: components/avatar/avatar.html [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    <link
+      href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="shortcut icon"
+      href="../../assets/favicon/favicon-red-black.ico"
+      type="image/x-icon"
+    />
+    <link rel="stylesheet" href="../../css/main.css" />
```
Comment (shivampandey0) [inline components/avatar/avatar.html:22]: The goal is to create Component Library, try putting all the sub-stylesheets into a single stylesheet using import and create a single CSS link in HTML file.

Refer to examples given in PRD on LMS

**Review by shivampandey0 — COMMENTED** [review]
Added some suggestions.

File: components/avatar/avatar.css (hunk 1 above)
Comment (omgaikwad) [reply components/avatar/avatar.css]: Thank you for suggestion, I will change it to rem.

File: components/avatar/avatar.html (hunk 2 above)
Comment (omgaikwad) [reply components/avatar/avatar.html:22]: I am aware of that, I am going to put all sub stylesheets into a single stylesheet in the end after completing all the components.

**Comment by omgaikwad** [comment]
> Your avatar.css file is looking good, But it is good to use the rem than px.

Thank you for your suggestion, I will change the code and use rem instead of px.

**Comment by omgaikwad** [comment]
> Naming Convention is good & comments readable. You can add border-color for variations one more It would be great if you have text-avatars just suggestions.

Thank you for the suggestions, I will try using border-color and see if it suits the requirements.

**Comment by omgaikwad** [comment]
Thank you for all your suggestions, I have made changes and I am ready to merge with the "dev" branch.

**Review by junaidshaikh-js — COMMENTED** [review]
Clean code! Good practices are followed. Ready to merge!

**Pull request merged**